examples/wip/viewport-animation/src/app.js

```diff
@@ -104,9 +104,8 @@ export default class App extends Component {
         viewportState={MapState}
         {...viewport}
         onViewportChange={this._onViewportChange.bind(this)}
-        transitionInterpolator={viewportFlyToInterpolator}
+        transitionInterpolator={new ViewportFlyToInterpolator()}
```

Comment:
Doesn't this create a new instance on every render?

Reply:
This is fairly cheap. I feel it makes the code easier to follow.